cypress-io/cypress
# fix: Correct pluralization in Spec Pattern

- Closes #21728 

### User facing changelog


- Display proper plural format for matches in the `Spec Patterns` section in `Project Settings`

### Additional details

NA

### Steps to test

- Open a project that has 1 spec
- Navigate to Spec Patterns under Project Settings
- Match count should read `1 Match`

### How has the user experience changed?


Before:
<img src="https://user-images.githubusercontent.com/25568640/179085257-696f0fd1-4ae7-486d-9a91-9cc2e0f64a57.jpg" alt="21728 - Before" height=400 />


After:
<img src="https://user-images.githubusercontent.com/25568640/179085390-686052ca-0ee4-4f3f-910a-4dd916a04cf2.jpg" alt="21728 - After" height=400 />


### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
